.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,318 @@
+name:  S3 tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - support/**
+    types: [opened, synchronize]
+    paths-ignore:
+      - '**/*.md'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      neofs-s3-gw_ref:
+        description: 'neofs-s3-gw ref. Default ref - latest master. Examples: v0.27.0, 8fdcc6d7e798e6511be8806b81894622e72d7fdc, branch_name'
+        required: false
+        default: ''
+
+permissions: write-all
+
+env:
+  S3_TESTS_CONFIG: s3tests.conf
+  S3_GW_IP: 192.168.130.82
+  S3_GW_PORT: 8080
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Set RUN_ID
+        env:
+          TIMESTAMP: ${{ steps.date.outputs.timestamp }}
+        run: echo "RUN_ID=${{ github.run_number }}-$TIMESTAMP" >> $GITHUB_ENV
+
+      - name: Checkout neofs-s3-gw repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-s3-gw
+          ref: ${{ github.sha }}
+          path: neofs-s3-gw
+
+      - name: Checkout s3-tests repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/s3-tests
+          ref: master
+          path: s3-tests
+
+      - name: Checkout neofs-dev-env repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-dev-env
+          ref: master
+          path: neofs-dev-env
+
+      - name: Checkout neofs-s3-dev-env repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-s3-dev-env
+          ref: master
+          path: neofs-s3-dev-env
+
+      - name: Download latest stable neofs-cli for uploading reports to NeoFS
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-node'
+          version: 'tags/v0.37.0'
+          file: 'neofs-cli-amd64'
+          target: 'neofs-node-stable/neofs-cli'
+
+      - name: Checkout neofs-testcases repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-testcases
+          ref: 'master'
+          path: neofs-testcases
+
+################################################################
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          cache: true
+          go-version: '1.20'
+      - run: go version
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - run: python --version
+
+################################################################
+      - name: Get TAG for docker images
+        run: |
+          echo "CURRENT_TAG=$( make version | sed 's/^v//' )" >> $GITHUB_ENV
+        working-directory: neofs-s3-gw
+
+################################################################
+
+      - name: Build neofs-s3-gw docker image
+        run: |
+          cp .github/test-env .env #looks ugly, but I want to try
```

Comment:
Does make require an .env file?

Reply:
You're right, it doesn't

Reply:
> You're right, it doesn't

If not needed, then why `cp .github/test-env .env` ?

Reply:
Now, I removed it